mods/cnc/maps/cnc64gdi01/map.yaml

```diff
@@ -0,0 +1,823 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: C&C64 Special Ops - GDI 1
+
+Description: Nod is experimenting on civilians with Tiberium. Use the commando to take out the SAM sites surrounding the dropoff area. With the SAMs gone you will then get an airstrike.  Take out the Obelisk and an MCV will be delivered to help you to locate and destroy the biochem facility.
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 64,64
+
+Bounds: 2,9,48,53
+
+Visibility: MissionSelector
+
+Type: Campaign
+
+Videos:
+	GameStart: obel.vqa
+	GameWon: orcabomb.vqa
+	GameLost: cutout.vqa
+
+Options:
+	Crates: False
+	Creeps: False
+	Fog: True
+	Shroud: True
+	AllyBuildRadius: False
+	FragileAlliances: False
+	StartingCash: 10000
+	ConfigurableStartingUnits: False
+	ShortGame: False
+
+Players:
+	PlayerReference@Nod:
+		Name: Nod
+		Faction: nod
+		ColorRamp: 3,255,127
+		Allies: Nod
+		Enemies: GDI, Creeps
+	PlayerReference@GDI:
+		Name: GDI
+		AllowBots: False
+		Playable: True
+		Required: True
+		LockFaction: True
+		Faction: gdi
+		LockColor: True
+		ColorRamp: 31,222,183
+		LockSpawn: True
+		LockTeam: True
+		Allies: GDI
+		Enemies: Nod, Creeps
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: gdi
+	PlayerReference@Creeps:
```

Comment:
You can remove this player. You set `Creeps: False` in the map options above, so it isn't needed.